src/fuzzfetch/fetch.py

```diff
@@ -109,6 +109,17 @@ def get_url(url):
     return data
 
 
+def resolve_url(url):
+    """Resolve requested URL"""
+    try:
+        data = HTTP_SESSION.head(url)
+        data.raise_for_status()
+    except requests.exceptions.RequestException as exc:
+        raise FetcherException(exc) from None
+
+    return data
```

Comment:
Minor nit but do we need to return anything here?

Reply:
I thought about removing this line, but since we're already resolving that all the paths work, I thought it might be useful in the future if we wanted to add parallel downloads. The headers would contain Content-Length, so we could use that to simplify the download function where we check the headers before entering the copy loop.